Pokemon Unity/Assets/Scripts2/Pokemon/Pokemon.cs

```diff
@@ -1595,6 +1664,11 @@ public string Mail
             }
             //set { mail = value; }
         }
+
+        public Item.Item.Mail GetMail()
```

Comment:
Is this supposed to be used for the Serialized class you made?

Reply:
Yes, I was trying out to convert Item.Item.Mail to Items (int) to save it inside Pokemon. It failed and I forgot to remove the function, should I remove it?